src/Microsoft.Health.Dicom.Web/Program.cs

```diff
@@ -31,13 +27,6 @@ public static void Main(string[] args)
                         tokenProviderConnectionString = $"RunAs=App;AppId={userAssignedAppId}";
                     }
 
-                    if (!string.IsNullOrEmpty(keyVaultEndpoint))
```

Comment:
Would this break userApp scenario? I see `tokenProviderConnectionString`  is not used anywhere.

Reply:
Oh yes, I should remove that too. I am unsure if we need these lines though. They looked to have been used mostly for CI. Perhaps @brandonpollett  has some thoughts on this whether we should keep it